libc/src/__support/FPUtil/double_double.h

```diff
@@ -73,6 +73,26 @@ template <size_t N = 27> LIBC_INLINE constexpr DoubleDouble split(double a) {
   return r;
 }
 
+// Helper for non-fma exact mult where the first number is already splitted.
+template <bool NO_FMA_ALL_ROUNDINGS = false>
```

Comment:
Having the double negation here adds slight cognitive burden to reading this. Consider making this:

```suggestion
template <bool FMA_ALL_ROUNDINGS = true>
```
Then flipping the condition below.

Reply:
Changed the template to pass the splitting constant explicitly.